src/scf/potential_mixing.jl

```diff
@@ -0,0 +1,345 @@
+using Statistics
+
+# Quick and dirty Anderson implementation ... lacks important things like
+# control of the condition number of the anderson matrix.
+# Not particularly optimised. Also should be moved to NLSolve ...
+function AndersonAcceleration(;m=Inf)
```

Comment:
lower case for function names

Reply:
or why not make the state explicit? anderson_acceleration!(Vs, PfVs, Vn, alphan, PfVn)?

Reply:
I would also make the names fully generic and use the same conventions as in scf_solvers.jl: x for iterates, f for residuals